.github/workflows/publish.yml

```diff
@@ -6,8 +6,14 @@ on:
         description: The site or venue that this project is being submitted to
         required: true
         type: string
+      collection:
```

Comment:
We could make this required? but it might be a little confusing for a venue with a "default" collection. I think I like optional and we put work into improving error messaging.

Reply:
Agree!